Fix unappropriated version of glib and gio.
For example polkitunixsession.c GInitTableIface interface and initable_iface_init are just available since 2.21.4 version of gio.
see https://bugs.gentoo.org/show_bug.cgi?id=279271 .
And related to NEWS file into tarball at least eggdbus 0.5 is needed.

 configure.ac |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0857323..b115539 100644
--- a/configure.ac
+++ b/configure.ac
@@ -118,11 +118,11 @@ if test "x$GCC" = "xyes"; then
   changequote([,])dnl
 fi
 
-PKG_CHECK_MODULES(GLIB, [glib-2.0 >= 2.14.0 gio-2.0 >= 2.14.0])
+PKG_CHECK_MODULES(GLIB, [glib-2.0 >= 2.21.4 gio-2.0 >= 2.21.4])
 AC_SUBST(GLIB_CFLAGS)
 AC_SUBST(GLIB_LIBS)
 
-PKG_CHECK_MODULES(EGG_DBUS, [eggdbus-1 >= 0.4])
+PKG_CHECK_MODULES(EGG_DBUS, [eggdbus-1 >= 0.5])
 AC_SUBST(EGG_DBUS_CFLAGS)
 AC_SUBST(EGG_DBUS_LIBS)
 
